components/trello/card-updates.js

```diff
@@ -0,0 +1,53 @@
+const spotify = require("https://github.com/PipedreamHQ/pipedream/components/trello/trello.app.js");
+
+module.exports = {
+  name: "Card Updates",
+  description: "Emits an event for each update to a Trello card.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    trello,
+    cardId: {
```

Comment:
We should use async options to list the cards available to the auth user

Reply:
I think it would also make sense for this prop to be an array of strings, i.e.

```
type: "string[]`
```

That way a user can watch multiple cards for updates without having to create a source per card.